src/compounds/BlankRequestForm/AdditionalInfo.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react'
+import { Card, Form } from 'react-bootstrap'
+import PropTypes from 'prop-types'
+
+const AdditionalInfo = ({ updateRequestForm }) => {
+  const [showProposalDate, setShowProposalDate] = useState(true)
+  const today = new Date().toISOString().slice(0, 10)
+  const handleChange = (value) => {
+    updateRequestForm(value, 'proposedDeadline')
+  }
+
+  return (
+    <Card className='mb-4'>
+      <Card.Header className='h3'>Additional Information</Card.Header>
+      <Card.Body>
+        {showProposalDate
+          && (
+            <Form.Group
+              className='mb-3'
+              controlId='required-by'
+            >
+              <Form.Label>Proposals Required By:</Form.Label>
+              <Form.Control
+                type='date'
+                min={today}
+                placeholder='Proposals Required By'
+                disabled={showProposalDate === false}
+                onChange={showProposalDate && ((e) => handleChange(e.target.value))}
+              />
+            </Form.Group>
+          )}
+        <Form.Group
+          className='mb-3'
+          controlId='open-proposal-due-date'
+        >
+          <Form.Check
+            type='checkbox'
+            label='Proposals can be submitted at any time.'
+            onChange={() => {
+              setShowProposalDate(!showProposalDate)
+              if (showProposalDate) handleChange(null)
+            }}
+          />
+        </Form.Group>
+        {/* TODO(summercook): figure out how these attachments will be saved. Currently we have an open question in slack */}
```

Comment:
the length of this comment throws a lint error. please run lint to resolve all errors.